### ISSUE TITLE
generated streaming method for nodejs-iot is missing preamble

### ISSUE THREAD

**Issue by crwilcox** [googleapis/gapic-generator#2152]
On our methods we have a bit of options setting before invoking. One method on the iot lib is missing this. https://github.com/googleapis/nodejs-iot/pull/5/commits/cf063192865d47fe4231438ba4382d0b06ab2c29

```  listDeviceRegistriesStream(request, options) {
     options = options || {};
+    options.otherArgs = options.otherArgs || {};
+    options.otherArgs.headers = options.otherArgs.headers || {};
+    options.otherArgs.headers[
+      'x-goog-request-params'
+    ] = gax.routingHeader.fromParams({
+      parent: request.parent,
+    });
 
     return this._descriptors.page.listDeviceRegistries.createStream(
       this._innerApiCalls.listDeviceRegistries,
```

**Comment by vchudnov-g** [comment]
@alexander-fenster ping

**Comment by vchudnov-g** [comment]
@alexander-fenster ping

**Comment by alexander-fenster** [comment]
P2, since Yoshi has a workaround in synth. I will take a look if it's an easy fix in the snip files.

**Comment by eoogbe** [comment]
@alexander-fenster Any updates?

**Comment by alexander-fenster** [comment]
It's fixed across all languages in #2644.
